kqueen/engines/base.py

```diff
@@ -187,6 +187,9 @@ def get_parameter_schema(cls):
         Returns:
             dict:  Returns ``self.parameter_schema`` in default, but can be overridden.
         """
+        if not hasattr(cls, 'parameter_schema'):
+            raise NotImplementedError('"parameter_schema" attribute should be provided in the '
```

Comment:
add logger before raising, coz this error does not traced on log by default
try to do:
```
msg = xxx
logger.debug(msg)
raise exception(msg)
```

Reply:
it is logged where we catch it, i've already added logger there